### PR TITLE
Proper implementation of check if current and new password are same w…

### PR DESCRIPTION
**Pull request by DravitLochan**
…hile changing user password

Fixes issue #385 

Changes: removed simple .equals() method to match current and new password. Now servlet matches hash values making the check more robust.

### CODE REVIEW

**Comment by DravitLochan** [comment]
@dynamitechetan @saurabhjn76 @madhavrathi @jyoth1raditya Please review!

**Comment by DravitLochan** [comment]
@dynamitechetan @saurabhjn76 @madhavrathi @jyoth1raditya Please review!

File: src/ai/susi/server/api/aaa/PasswordChangeService.java [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- *  Lesser General Public License for more details.
- *
- *  You should have received a copy of the GNU Lesser General Public License
- *  along with this program in the file lgpl21.txt
- *  If not, see <http://www.gnu.org/licenses/>.
+ * PasswordResetService
+ * Copyright 6/7/17 by Dravit Lochan, @DravitLochan
+ * <p>
```
Comment (dynamitechetan) [inline src/ai/susi/server/api/aaa/PasswordChangeService.java]: is this <p> required?

Comment (DravitLochan) [inline src/ai/susi/server/api/aaa/PasswordChangeService.java]: not required. Removed!

**Pull request merged**